griptape/artifacts/__init__.py

```diff
@@ -14,5 +14,5 @@
     "TextArtifact",
     "BlobArtifact",
     "CsvRowArtifact",
-    "ListArtifact"
+    "ListArtifact",
```

Comment:
Can we disable this rule? I think this can be sketchy considering how Python interprets tuples

Reply:
Disabled